jdi-light-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/MultiSelectorSteps.java

```diff
@@ -0,0 +1,69 @@
+package com.epam.jdi.bdd.stepdefs;
+
+import com.epam.jdi.light.ui.html.complex.MultiSelector;
+import com.epam.jdi.tools.LinqUtils;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+
+import java.util.List;
+
+import static com.epam.jdi.bdd.Utils.getUI;
+import static org.hamcrest.Matchers.*;
+import static org.testng.Assert.assertEquals;
+
+public class MultiSelectorSteps {
+
+    @When("^I select \"([^\"]*)\" with value \"([^\"]*)\"$")
```

Comment:
add : to the end of all steps where input as gherkin table

Reply:
In this step I don't see a gherkin table, but I have ":" in other steps